liblangutil/ErrorReporter.cpp

```diff
@@ -94,6 +94,16 @@ bool ErrorReporter::checkForExcessiveErrors(Error::Type _type)
 		if (m_warningCount >= c_maxWarningsAllowed)
 			return true;
 	}
+	else if (_type == Error::Type::Info)
+	{
+		m_infoCount++;
+
+		if (m_infoCount == c_maxInfosAllowed)
+			m_errorList.push_back(make_shared<Error>(2833_error, Error::Type::Warning, "There are more than 256 infos. Ignoring the rest."));
```

Comment:
Might be annoying if having > 256 infos increases the severity and makes them non-ignorable.
```suggestion
			m_errorList.push_back(make_shared<Error>(2833_error, Error::Type::Info, "There are more than 256 infos. Ignoring the rest."));
```